### PR TITLE
Dahyeon Kang memeber profile added

### LINKED ISSUE #5573: [Community] Member Profile: Dahyeon Kang
**Current Behavior**

@dusdjhyeon has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @dusdjhyeon as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @dusdjhyeon
- Twitter: 
- LinkedIn: 
- Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by iArchitSharma**
**Description**

This PR fixes #5573 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 57d16916aadd0d938b580422dd8a5abb12809a08 at: https://666bc152048ee00c78e87cbe--layer5.netlify.app

File: src/collections/members/dahyeon-kang/index.mdx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Dahyeon Kang
+position: Contributor
+image_path: ./Dahyeon-Kang.webp
+github: dusdjhyeon
+twitter: dusdjhyeon
+linkedin: 다현-강-27a578297
+layer5: 2662825c-2bc7-4dea-a7ef-91269d334f71
+location: India
```
Comment (sudhanshutech) [inline src/collections/members/dahyeon-kang/index.mdx:9]: ?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: src/collections/members/dahyeon-kang/index.mdx (hunk 1 above)
Comment (iArchitSharma) [reply src/collections/members/dahyeon-kang/index.mdx:9]: oops

Comment (iArchitSharma) [reply src/collections/members/dahyeon-kang/index.mdx]: fixed*

**Pull request merged**